diagnostic.md

```diff
@@ -91,20 +132,15 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+The font family for .specialDiv is cursive,sans-serif and color is yellow
```

Comment:
The font-family is only cursive because inline styling overrides the stylesheet.